src/Tool-CriticBrowser/CBSelectPackageBrowser.class.st

```diff
@@ -0,0 +1,174 @@
+"
+I'm using to selectPakage to analyse
+
+    Instance Variables
+	addTestCaseClassesCheckBox:		<SpCheckBoxPresenter>
+	removeTestPackageButton:		<SpButtonPresenter>
+		
+	testPackagesToRemove:		<OrderedCollection>
+			that contains all selected package
+
+"
+Class {
+	#name : #CBSelectPackageBrowser,
+	#superclass : #CBSelectBrowser,
+	#instVars : [
+		'addTestCaseClassesCheckBox',
+		'removeTestPackageButton',
+		'testPackagesToRemove'
+	],
+	#category : #'Tool-CriticBrowser-Parts'
+}
+
+{ #category : #specs }
+CBSelectPackageBrowser class >> defaultSpec [
+	^ SpBoxLayout newVertical
+		add:
+			(SpPanedLayout newHorizontal
+				add: #toSelectPresenter;
+				add: #selectedPresenter;
+				yourself);
+		add:
+			(SpBoxLayout newHorizontal
+				add: #removeTestPackageButton;
+				add: #addTestCaseClassesCheckBox;
+				yourself)
+			withConstraints: [ :contraint | contraint height: 25 ];
+		add:
+			(SpBoxLayout newHorizontal
+				add: #addButton;
+				add: #addAllButton;
+				add: #removeButton;
+				add: #removeAllButton;
+				add: #nextButton;
+				yourself)
+			withConstraints: [ :contraint | contraint height: 25 ];
+		yourself
```

Comment:
This method has a lot of things in common with the one in the superclass. Can't it be refactored to use super?

Reply:
in my case, I think that using refactoring is a bad idea because in CBSelectPackageBrowser defaultSpec I insert 2 widgets between 2 group of widget.
in case of adding at the start or at the end of the specDefault we must use refactoring